src/client.ts

```diff
@@ -134,4 +134,19 @@ export class Client {
       body: form,
     });
   }
+
+  /**
+   * Make a DELETE request to the specified Imgur API endpoint with form data
+   *
+   * @param endpoint An Imgur API endpoint
+   * @param method Default method is DELETE. For certain endpoints like unblocking users you
+   * need to pass a GET.
+   * @returns A JSON response object
+   */
+
+  delete(endpoint: string, method: string = 'DELETE'): Promise<any> {
```

Comment:
adding `delete()` is great, but I think it would be cleaner to use `client.get()` to send the `GET` request in the `unblock` module, rather than update `delete()` to handle multiple http verbs

Reply:
You're right. 👍